.github/workflows/crates-release.yml

```diff
@@ -0,0 +1,22 @@
+# See https://crates.io/docs/trusted-publishing
+name: Publish to crates.io
+on:
+  push:
+    tags: ['v*']  # Triggers when pushing tags starting with 'v'
+jobs:
+  publish:
+    runs-on: ubuntu-24.04
+    permissions:
+      id-token: write     # Required for OIDC token exchange
+    steps:
+    - uses: actions/checkout@v6
+    - uses: rust-lang/crates-io-auth-action@v1
+      id: auth
+    - run: |
+        for crate in bootc-internal-utils bootc-internal-blockdev; do
```

Comment:
I think we also need to add a step to our release workflow which bumps the versions of these crates right?